hypothesis-python/tests/py3/test_lookup.py

```diff
@@ -599,3 +599,55 @@ def test_bytestring_is_valid_sequence_of_int_and_parent_classes(type_):
         st.from_type(typing.Sequence[type_]),
         lambda val: isinstance(val, typing.ByteString),
     )
+
+
+def supports_protocol(protocol, inst):
+    try:
+        # in python 3.8 this works
+        return isinstance(inst, protocol)
+    except TypeError:
+        # in python < 3.8 Protocols cannot be used with isinstance
+        # so we do this which is less nice...
+        return issubclass(type(inst), protocol)
```

Comment:
I would just write `issubclass(type(inst), protocol)` inline - we only use it in two places, and we won't be dropping Python 3.7 for a long time yet.

Reply:
yeah is overkill .... will change